server/SocketManager/socketmanager.js

```diff
@@ -1,11 +1,14 @@
 let sockets = {};
+let socket_user_map = {};
```

Comment:
The variable `socket_user_map` is introduced but lacks a clear description of its purpose. Consider adding a comment above its declaration to explain its role in the code.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*